packages/@pollyjs/core/src/defaults/config.js

```diff
@@ -16,10 +16,10 @@ export default {
   logging: false,
 
   recordIfMissing: true,
-  recordIfExpired: false,
   recordFailedRequests: false,
 
   expiresIn: null,
+  expiryStrategy: 'replay',
```

Comment:
This should be defaulted to `warn`.

Reply:
Might also be worth using the `EXPIRY_STRATEGIES` import